### DISCUSSION TITLE
Docs: Is there a list of available plugins with descriptions of what they do?

### DISCUSSION THREAD

**Discussion by mhkeller** [perspective-dev/perspective#2398] [Q&A] [answered]
I see in the [Evictions](https://perspective.finos.org/block/?example=evictions) example, if you load [@finos/perspective-viewer-openlayers](https://www.npmjs.com/package/@finos/perspective-viewer-openlayers) you get access to the Map Scatter view, which is really neat. Is there a list of other available extras like this? I don't see any other likely candidates in the [packages](https://github.com/finos/perspective/tree/master/packages) directory but figured I'd ask.

**Comment by timkpaine** [comment, marked as answer]
There is no centralized registry of plugins, but included in perspective:
- Map Scatter - Openlayers (https://github.com/finos/perspective/blob/master/packages/perspective-viewer-openlayers/src/js/views/map-view.js#L143)
- Datagrid - Datagrid 
- All other charts - d3fc (https://github.com/finos/perspective/blob/master/packages/perspective-viewer-d3fc/src/js/charts/charts.js#L26)

Then of course anyone can write their own: example summary plugin https://github.com/perspective-community/perspective-summary
